src/ignitus-Shared/ignitus-DesignSystem/ignitus-Atoms/ignitus-defaultIconInput/Components/defaultIconInput/index.tsx

```diff
@@ -15,7 +15,7 @@ export const DefaultIconInput = ({
 
   useEffect(() => {
     handleChange(value);
-  }, [value]);
+  }, [handleChange, value]);
```

Comment:
@Dishebh please test you pr for such cases, `handleChange` is changing every time on each keypress.

`or one of the dependencies changes on every render`

![Screenshot 2020-07-11 at 01 40 11](https://user-images.githubusercontent.com/13464678/87210924-99c96c00-c317-11ea-8fcf-59407362f28a.png)

It was freezing the ignitus react-app, I reverted this change now.

Reply:
Sure, will take care that it won't repeat. Thanks